lib/File/ShareDir.pm

```diff
@@ -435,6 +435,13 @@ else
 {
     ## no critic (ErrorHandling::RequireCheckingReturnValueOfEval)
     eval <<'END_OF_BORROWED_CODE';
+
```

Comment:
NOPE. This is internal and must not be used from outside.

Reply:
Is there any way hide this error?

Reply:
Yes, make sure you have List::MoreUtils 0.428 installed.

Reply:
Indeed it hides the error.

Reply:
Which error. Can you show me the error?
I've tested it with Perls which have no LMU 0.428 installed and they don't show any error

Reply:
The author POD test fail with a `firstres` is undocumented error. There is an eval 'use LMU 0.428' in the lines above it (line 430) which then imports it.

Reply:
Oh, there were two errors in the description.

Yeah - for that failure, ignoring the 'firstres' in the test would be right.